### PR TITLE
Rework default volume settings

### PR DESCRIPTION
**Pull request by lambdcalculus**
Addresses #892 by making the client remember the last volume settings from the sliders. This makes the volume settings in the options dialog unnecessary, so those were removed.

### CODE REVIEW

**Review by stonedDiscord — APPROVED** [review]
The destructor is made for this, it should work very well

**Review by Salanto — CHANGES_REQUESTED** [review]
Approval from me when the page is corrected.

There is the theoretical issue that multi-client scenarios with different volume per client means the last one closed is the one whose volume is saved, but that boils down to a lack of synchronisation between clients, so not an issue to be adressed here.

File: resource/ui/options_dialog.ui [hunk 1]
```diff
@@ -23,7 +23,7 @@
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>2</number>
```
[the remark this answers is not in the document]
Comment (stonedDiscord) [reply resource/ui/options_dialog.ui:26]: good catch

[the remark this answers is not in the document]
Comment (Salanto) [reply resource/ui/options_dialog.ui:26]: Happens to me all the time, lol

**Review by Salanto — APPROVED** [review]

**Review by Salanto — CHANGES_REQUESTED** [review]
Actually, hold up, I just checked Courtrooms code and apparently we still use the hardcoded volumes of the slider when populating the volume.

We'd need to adjust that too in the constructor.

**Comment by lambdcalculus** [comment]
Does it? From what I can see, it's calling `Options::musicVolume()` which gets the default music volume from the config.ini, or sets it to 50 if `default_music` is missing. Same for sfx and blips. Can you point me to what's wrong?

For the record, I did test this and it worked, so, yeah.

**Review by Salanto — APPROVED** [review]
Nvm, I am borderline stupid. Never do PR review while @ work

**Pull request merged**